src/Capgemini.PowerApps.PackageDeployerTemplate/Services/SlaActivatorService.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Capgemini.PowerApps.PackageDeployerTemplate.Adapters;
+using Capgemini.PowerApps.PackageDeployerTemplate.Extensions;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Capgemini.PowerApps.PackageDeployerTemplate.Services
+{
+    public class SlaActivatorService
```

Comment:
As per previous - it also looks like we're only activating SLAs which we're saying we want as default and these are two separate issues.

Reply:
I'll separate those to respective methods.